openedx/core/djangoapps/theming/template_loaders.py

```diff
@@ -41,13 +41,7 @@ def get_template_sources(self, template_name, template_dirs=None):
         if isinstance(theme_dirs, list):
             template_dirs = theme_dirs + template_dirs
 
-        for template_dir in template_dirs:
-            try:
-                yield safe_join(template_dir, template_name)
-            except SuspiciousFileOperation:
-                # The joined path was located outside of this template_dir
-                # (it might be inside another one, so this isn't fatal).
-                pass
+        return list(super(ThemeFilesystemLoader, self).get_template_sources(template_name, template_dirs))
```

Comment:
Seems good to remove this ignoring of the exception...

Reply:
The same try/except is still in the superclass implementation that's called here; I just had to change it like this because this method changed in 1.10 from a `unicode` generator to an `Origin` object generator.